src/client/src/ui/spotTile/components/SpotTile.tsx

```diff
@@ -38,7 +39,22 @@ export default class SpotTile extends PureComponent<Props, State> {
 
   updateNotional = (notional: string) => this.setState({ notional })
 
-  executeTrade = (direction: Direction) => this.props.executeTrade(direction, numeral(this.state.notional).value())
+  createTrade = (direction: Direction) => {
```

Comment:
Could we not pull some of this to the mapToDispacth in the container?

Reply:
I don't believe you would have access to `spotTileData` in `mapToDispatch`

Reply:
OK